pages/test_analytics/flaky_test_assignment.md

```diff
@@ -34,3 +34,8 @@ When an assigned test has not flaked in more than 7 days, it is moved to the **O
 
 <%= image "outdated-assignments.png", width: 1960/2, height: 630/2, alt: "Flaky test page showing team assignments" %>
 
+## Weekly flaky test mailer
```

Comment:
The word mailer doesn't feel right to me here, maybe "Weekly flaky test summary report"? Or "Weekly flaky test summary"

Reply:
Cool, will change to Weekly flaky test summary